Sources/Hummingbird/Middleware/MetricsMiddleware.swift

```diff
@@ -31,12 +31,12 @@ public struct MetricsMiddleware<Context: RequestContext>: RouterMiddleware {
                 // need to create dimensions once request has been responded to ensure
                 // we have the correct endpoint path
                 let dimensions: [(String, String)] = [
-                    ("hb_uri", context.endpointPath ?? request.uri.path),
-                    ("hb_method", request.method.rawValue),
+                    ("uri", context.endpointPath ?? request.uri.path),
+                    ("method", request.method.rawValue),
                 ]
-                Counter(label: "hb_requests", dimensions: dimensions).increment()
+                Counter(label: "hb.requests", dimensions: dimensions).increment()
```

Comment:
```suggestion
                Counter(label: "http.server.active_requests", dimensions: dimensions).increment()
```

Reply:
As I understand it that is the number of current active requests, not the total number of requests which is what `hb.requests` is measuring